src/components/v5/common/ActionSidebar/partials/NonVerifiedMembersSelect/NonVerifiedMembersSelect.tsx

```diff
@@ -0,0 +1,164 @@
+import clsx from 'clsx';
+import { utils } from 'ethers';
+import React, { type FC } from 'react';
+import { useController, useFormContext } from 'react-hook-form';
+
+import { useAdditionalFormOptionsContext } from '~context/AdditionalFormOptionsContext/AdditionalFormOptionsContext.tsx';
+import { useMobile } from '~hooks/index.ts';
+import useRelativePortalElement from '~hooks/useRelativePortalElement.ts';
+import useToggle from '~hooks/useToggle/index.ts';
+import useUserByAddress from '~hooks/useUserByAddress.ts';
+import { formatText } from '~utils/intl.ts';
+import Button from '~v5/shared/Button/Button.tsx';
+import SearchSelect from '~v5/shared/SearchSelect/index.ts';
+import UserAvatar from '~v5/shared/UserAvatar/index.ts';
+
+import { useVerifiedMembersSelect } from './hooks.ts';
+import { type NonVerifiedMembersSelectProps } from './types.ts';
+
+const displayName =
+  'v5.common.ActionsContent.partials.NonVerifiedMembersSelect';
+
+const NonVerifiedMembersSelect: FC<NonVerifiedMembersSelectProps> = ({
+  name,
+  fieldArrayMethods,
+}) => {
+  const {
+    field,
+    fieldState: { error },
+  } = useController({
+    name,
+  });
+  const isError = !!error;
+  const { usersOptions } = useVerifiedMembersSelect();
+  const [
+    isUserSelectVisible,
+    {
+      toggle: toggleUserSelect,
+      toggleOff: toggleOffUserSelect,
+      registerContainerRef,
+    },
+  ] = useToggle();
+  const { user: userByAddress, loading: userByAddressLoading } =
```

Comment:
is there an option to store the entire member object when selecting? now we're fetching the members, storing their address in form values and then refetching them again